.gitignore

```diff
@@ -59,3 +59,4 @@ analyze.py
 # Editor backup files
 *~
 .*.sw?
+.vscode/
```

Comment:
Instead, you should run
```
git config --global --add core.excludesfile ~/.gitignore_global
```
and create a `~/.gitignore_global` file that does this for all repos you use!